git-pr.toml

```diff
@@ -2,13 +2,15 @@
 url = "localhost"
```

Comment:
I'm also not fully happy with that parameter yet :smile:

I currently add `-p 2222 pr.in0rdr.ch` to ensure users grok that they need to configure the port as well.

However, this is trouble when viewing the PRs on the command line, for instance, using `ssh pr.in0rdr.ch pr summary 1`:

```
Info
====
URL: https://-p 2222 pr.in0rdr.ch/prs/1
```

It might break the help texts in other parts of the application too.

On that note, I dislike `pr summary 1`. How about `pr show 1` (or `get`), summary is so painfully long :smile: 

Otherwise, I really enjoy that project a lot, thank you so much!

Reply:
I solved the issue in my case by just using the standard port 22. Everything else is too much to ask from the user anyways :smile: